### DISCUSSION TITLE
GitHub Codespaces v1.9.1

### DISCUSSION THREAD

**Discussion by marcelokuhl** [community/community#18930] [Codespaces]
Após atualizar o GitHub Codespaces para a versão v.1.9.1, não consigo mais compilar usando o CS50 (code50),,,
Como desfazer  a atualização? O makefile parece que foi alterado. 
OBS.: os programas funcionavam na versão anterior.
![image](https://user-images.githubusercontent.com/68967909/174493641-2844e500-ef20-4979-bc19-b179aa47aea4.png)

![image](https://user-images.githubusercontent.com/68967909/174493699-98842aa0-83f7-412c-bf20-370aa1cd7f9a.png)

**Comment by dmgardiner25** [comment]
@marcelokuhl You can downgrade to previous versions of Codespaces by going to the extensions tab in VS Code, clicking the gear for "GitHub Codespaces", and then clicking "Install Another Version...".
<img width="500" alt="image" src="https://user-images.githubusercontent.com/22037769/174670335-db973d42-4e97-4f6b-a93d-904a93b687f0.png">

However, I wouldn't think changes to the CS50 makefile would be affected by the Codespaces extension in any way. Please let me know if downgrading fixes your issues and if not, I would try reaching out to the CS50 team for help.

**Reply by marcelokuhl** [reply]
Muito obrigado. Mas hoje pela manha (20/06/2022) apareceu uma atualização da extensão (automática) e apóis fazê-la o compilador C, voltou a funcionar.  Mas uma vez muito obrigado pela atenção !!!
